docker_cli/_cli.py

```diff
@@ -0,0 +1,327 @@
+
+    setup_all_parser = setup_subparsers.add_parser(
+        "all", parents=[cuda_run_parse, no_cache_parse],
+        help="Set up the full docker image stack.",
+        formatter_class=help_formatter
+    )
+    setup_all_parser.add_argument(
+        "--tag", "-t", default="setup", type=str,
+        help='The sub-prefix of the docker images to be created. Generated images will '
+        f'have tags: "{prefix}-[TAG]-*". Default: "setup"'
+    )
+    setup_all_parser.add_argument(
+        "--base", "-b", default="oraclelinux:8.4", type=str,
+        help='The name of the parent docker image. Default: "oraclelinux:8.4"'
+    )
+    setup_all_parser.add_argument(
+        "--runtime-env-file", default="runtime-spec-file.txt", type=str,
+        help='The location of the runtime spec-file. Default: "runtime-spec-file.txt"'
+    )
+    setup_all_parser.add_argument(
+        "--dev-env-file", default="dev-spec-file.txt", type=str,
+        help='The location of the dev spec-file. Default: "dev-spec-file.txt"'
+    )
+
+    setup_init_parser = setup_subparsers.add_parser(
+        "init", parents=[no_cache_parse],
+        help="Set up the configuration image.",
+        formatter_class=help_formatter
+    )
+    setup_init_parser.add_argument(
+        "--base", "-b", default="oraclelinux:8.4", type=str, required=True,
+        help='The name of the parent docker image. Default: "oraclelinux:8.4"'
+    )
+    _add_tag_argument(parser=setup_init_parser, default="init")
+
+    setup_cuda_parser = setup_subparsers.add_parser(
+        "cuda",
+        help="Set up a CUDA image. Designate dev or runtime.",
+        formatter_class=help_formatter
+    )
+
+    cuda_subparsers = setup_cuda_parser.add_subparsers(dest="cuda_subcommand")
+    setup_cuda_runtime_parser = cuda_subparsers.add_parser(
+        "runtime", parents=[setup_parse, cuda_run_parse, no_cache_parse],
+        help="Set up the CUDA runtime image.",
+        formatter_class=help_formatter
+    )
+    _add_tag_argument(parser=setup_cuda_runtime_parser, default="cuda-runtime")
+
+    setup_cuda_dev_parser = cuda_subparsers.add_parser(
+        "dev", parents=[setup_parse, no_cache_parse],
+        help="Set up the CUDA dev image.",
+        formatter_class=help_formatter
+    )
+    _add_tag_argument(parser=setup_cuda_dev_parser, default="cuda-dev")
+
+    setup_env_parser = setup_subparsers.add_parser(
+        "env",
+        help="Set up the runtime environment image. "
+             "Designate dev or runtime.",
+        formatter_class=help_formatter
+    )
+
+    env_subparsers = setup_env_parser.add_subparsers(dest="env_subcommand")
+    setup_env_runtime_parser = env_subparsers.add_parser(
+        "runtime", parents=[setup_parse, env_parse, no_cache_parse],
+        help="Set up the runtime conda environment image",
+        formatter_class=help_formatter
+    )
+    _add_tag_argument(parser=setup_env_runtime_parser, default="env-runtime")
```

Comment:
Similarly, let's change the default tags of these images to "conda-runtime" and "conda-dev" instead of "env-runtime" and "env-dev"

```suggestion
    _add_tag_argument(parser=setup_env_runtime_parser, default="conda-runtime")
```